test/smoke/src/areas/positron/console/consoleANSI.test.ts

```diff
@@ -0,0 +1,97 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (C) 2024 Posit Software, PBC. All rights reserved.
+ *  Licensed under the Elastic License 2.0. See LICENSE.txt for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+
+import { expect } from '@playwright/test';
+import { Application, Logger, PositronRFixtures } from '../../../../../automation';
+import { installAllHandlers } from '../../../utils';
+import { join } from 'path';
+
+export function setup(logger: Logger) {
+	describe('Console ANSI styling', () => {
+		// Shared before/after handling
+		installAllHandlers(logger);
+
+		describe('R - Console ANSI styling', () => {
+
+			before(async function () {
+				await PositronRFixtures.SetupFixtures(this.app as Application);
+				await this.app.workbench.positronLayouts.enterLayout('fullSizedPanel');
+			});
+
+			after(async function () {
+				const app = this.app as Application;
+				await app.workbench.positronLayouts.enterLayout('stacked');
+			});
+
+			it("R - Can produce clickable file links", async function () {
+				const app = this.app as Application;
+
+				// Can be any file on the workkspace. We use .gitignore as it's probably
+				// always there.
+				const fileName = '.gitignore';
+				const filePath = join(app.workspacePathOrFolder, fileName);
+				const inputCode = `cli::cli_inform("{.file ${filePath}}")`;
+
+				await expect(async () => {
+					await app.workbench.positronConsole.pasteCodeToConsole(inputCode);
+					await app.workbench.positronConsole.sendEnterKey();
+
+					// Locate the link and click on it
+					const link = app.workbench.positronConsole.getLastClickableLink();
+					await expect(link).toContainText(fileName, { useInnerText: true });
+
+					await link.click();
+					await app.code.wait(200);
```

Comment:
Is this wait (and all of them) needed? I would think the subsequent command would have enough of a timeout?

Reply:
I removed a few waits, but I'm not sure about the others as they were there before. Locally it's ok if I remove them..